### ISSUE TITLE
Sayma RTM clocking

### ISSUE THREAD

**Issue by hartytp** [sinara-hw/sinara#405]
There have been a lot of developments since we originally designed the clocking scheme for Sayma. These affect the optimal design for the RTM clock network. Currently this network is quite complicated and has a lot of muxes. Suggestion to simplify it (hence reduce cost, complexity, power consumption, number of muxes that need to be supported in gateware):

Edit: **my original suggestion is at the bottom of this issue. Assuming we drop the RFBP #498, I suggest the following**:
1. **Reference sources**: (a) SMA on RTM front pannel, (b) DRTIO clock from Si5324
  - both sources directly feed a single ADCLK948 mux (no LTC chip). Pullups/pulldowns added to mux, to set default clock input. Default clock input should be Si5324. Outputs of this mux go to:
    - 1 to HMC830 for DAC clock
    - 1 to a test connector (I'd prefer MMCX to UFL)
    - 1 to input of LO mux
2. **DAC clock**: only source is HMC830
  - HMC830 feeds HMC7043 directly. Outputs from this go to:
    - DACs
    - FPGA
    -  test connector (again, an MMCX please)
  - If we still have doubts about the reliability of the HMC830 then we should put a solder jumper between the HMC830 and the HMC7043, which allows the LO (see below) to be used as an external DAC clock input (I only expect this to be needed for debugging, so not worth an extra mux for this, but let's make the jumpers 0402 or larger).
3. **LO/REF for AFEs**: sources are (a) SMA on Sayma RTM front panel and, (b) and the reference clock from the reference mux (this is useful for AFE mezzanines including a synth to generate the LO).
  - These feed a single mux, which feeds the AFE mezzanines and a test connector
  - Default input of this mux, as set by pullups/pull-downs, is the reference clock

**Original suggestion**.
- [x] Remove support for DAC clock from backplane, front-panel and clock mezzanine. Only required source of DAC clock should be HMC830. (cf #407).
  - [x] Remove the DAC clock signal from the Radial RF BP connector
  - [x] Remove TR3 c603 C602 J64 and J65 and not have to document this path for getting the RF DAC backplane clock into the DACs (this path involves coax jumpers!).
  - [x] Remove the mux IC58 and associated components (just connect the HMC830 directly to the HMC7043)
  - Note that in the current design, sending the DAC clock down the RF backplane requires a pair of UFL jumper cables, which is a nasty hack!
  - This reduces the complexity of the gateware/software required to control Sayma RTM (fewer muxes) and simplifies Baikal quite a bit.

**LO**
- [x] only provide two options for LO distribution to Sayma: backplane and front-pannel. This requires a single ADCLK948 to mux and then fanout to the AFE mezzanines, which is a much lower level of complexity than the current design. This reduces cost, power consumption, hardware complexity, number of ICs to test, gateware/software complexity.
  - It's really nice to have a single LO input to Sayma, which is then supplied to the AFEs for upconverting etc. Having each upconverting AFE take its own LO input via the FP takes up a lot of FP space, which we don't really have, and relies on having a lot of external wiring and LO distribution. Although, since each mezzanine has its own FP, mezzanine designers are free to do this if they need (e.g. for a very high frequency LO if the designer doesn't want to just multiply the LO distributed to the LO).
  - AFAICT, providing the same LO to all Sayma in the same rack is fine. cf #407 so we don't need to provide the option of generating a LO locally on Sayma via a clock mezzanine

**Comment by hartytp** [comment]
Can someone remind me what REFSYNCIN is for on Sayma RTM, and whether we still need it? IIRC it's a fallback plan for synchronization, in case the M-Labs scheme doesn't work, so can be scrapped once that's working. Is that correct?

**Comment by hartytp** [comment]
@gkasprow It's been a while since I looked at the RF BP schematics, but can you remind me how the two connectors work. AFACIT:
- We have the ERNI connector (J74), which is currently only used for the 100MHz clock, u_RFB_REF_CLK_IN, as well as the synchronisation signal, REFSYNCIN.
- Additionally, we have a Radial R694252107 RF connector, J73, which supports 3 differential signals up to 6GHz. 
- Can we use the Radial connector to carry REF_CLK, DAC_CLK and LO, and scrap the ERNI connector? AFACIT, that would be sufficient for all the signals we want to use, and would save some cost + routing.

**Comment by hartytp** [comment]
@gkasprow Do we have any data on the performance (cross-talk and return-loss) of the ERNI connector as a function of frequency? If not, could you measure it for us, please?

This would be really helpful information, as it could potentially allow us to simplify things quite a bit:
- Currently the REC_CLK is distributed over both the ERNI and Radial connector. IIRC, this is because we were worried about reflections from the ERNI connector. If this is not a problem, then only distributing REF_CLK over the ERNI connector would allow us to:
  - uRFB_REF_IN signal (simplifies schematic + routing)
  - C667 C604 C605 R238 R239 R58 R59 TR2 
  - This allows us to remove a solder jumper + another clocking option, which would otherwise have to be explained properly in the documentation.
- If the ERNI connector is decent at RF frequencies up to a few GHz, we can consider using it to distribute the LO instead of the radial connector. 
  - This would allow us to DNP the radial connector, which is 5 times the price of the ERNI connector.
  -

**Comment by gkasprow** [comment]
@hartytp the difference between these connectors is how the traces are routed.
The ERNI_973028 is for differential, digital LVDS signals .
The RADIALL_R694252107 is for single ended, high isolation clock routing.  That's why DAC, reference and the LO signals are routed there. The idea was to use common clock distribution mezzanine in the Baikal module instead of in every AMC. The clock distribution module in the Baikal is the same as in Sayma.
So it is not the connector who is limiting factor here but the RF backplane design.

**Comment by hartytp** [comment]
> The ERNI_973028 is for differential, digital LVDS signals .
> The RADIALL_R694252107 is for single ended, high isolation clock routing.

Differential traces can be excellent for high-isolation clocks too (in fact, they're often preferred to single ended traces). Is the concern here that the routing of these differential traces is not done in a low-crosstalk fashion, so that they will provide worse performance than the Radial connector?

If the ERNI connector is bad for routing clock traces, then why do we have a clock trace routed on it? Can we just scrap that trace as well as the R58 R59 jumper?

Do we actually need the ERNI connector at all (we generate all the voltages we need locally using SMSPs AFACT), so can we just DNP it?

**Comment by gkasprow** [comment]
@hartytp sure, we can DNP them by default.
All questions about RF backplane should go to @sbouhabib who actually was involved in the design.

**Comment by hartytp** [comment]
Okay, I'd be interested to hear from @sbouhabib about the possibility of routing low-noise LOs through the ERNI connector.

But, assuming that we can't use the ERNI for clocks/LOs, I don't see any reason not to DNP it to save some money. We might as well also remove the REF_CLK line that connects to it as well as the solder jumpers to simplify the schematic a bit (at this point there are so many solder jumpers etc that the schematic is quite hard to follow, if we can simplify the design a bit, it makes the schematic much more understandable).

**Comment by gkasprow** [comment]
Most of the discussion here is not relevant. RTM backplane was removed.
